### PR TITLE
Fix slot based busy detection

### PR DESCRIPTION
**Pull request by Mashintime**
TNC will not go into busy state if slot 0 is not busy as variable gets clobbered.

### CODE REVIEW

File: tnc/modem.py [hunk 1]
```diff
@@ -1251,11 +1251,11 @@ def calculate_fft(self) -> None:
                         range_end = range[1]
                         # define the area, we are detecting busy state
                         #dfft = dfft[120:176] if TNC.low_bandwidth_mode else dfft[65:231]
-                        dfft = dfft[range_start:range_end]
+                        slotdfft = dfft[range_start:range_end]
                         # Check for signals higher than average by checking for "100"
                         # If we have a signal, increment our channel_busy delay counter
                         # so we have a smoother state toggle
-                        if np.sum(dfft[dfft > avg + 15]) >= 400 and not TNC.transmitting:
+                        if np.sum(slotdfft[slotdfft > avg + 15]) >= 400 and not TNC.transmitting:
```
Comment (Mashintime) [inline tnc/modem.py:1258]: @DJ2LS I'm not sure if the >= 400 should/needs to be adjusted since we're looking at a smaller sample size.  Thoughts?

Comment (DJ2LS) [reply tnc/modem.py:1258]: This needs to be carefully tested I think. We can reduce this step by step. What about reducing it to 200 as a first test?

Comment (Mashintime) [reply tnc/modem.py:1258]: 200 definitely seems better, but may need further refinement.

**Comment by DJ2LS** [comment]
@Mashintime sorry, I've added a broadcast related commit.

**Pull request merged**